playbooks/incident-handling.md

```diff
@@ -0,0 +1,458 @@
+  may be responsible.
+  [Roll back](https://github.com/artsy/hokusai/blob/master/docs/Command_Reference.md#how-to-do-a-rollback) the
+  culprit if necessary.
+- Start a video conference to communicate in real-time with other responders (see 1Password for shared Zoom account
+  credentials).
+- See
+  [Engineering Playbooks (Notion)](https://www.notion.so/artsy/Engineering-Playbooks-b655fe54c1ce4b35af342c9ed9a489ae)🔒
+  and [legacy wiki docs](https://github.com/artsy/potential/wiki)🔒 for solutions to common problems.
+- Datadog [monitors](https://app.datadoghq.com/monitors/triggered?q=),
+  [service traces](https://app.datadoghq.com/apm/services), and
+  [dashboards](https://app.datadoghq.com/dashboard/lists) can illuminate the source and scale of latency or errors.
+- Error details are [captured in Sentry](https://sentry.io/organizations/artsynet/issues).
+- Logs are searchable [in Papertrail](https://papertrailapp.com/dashboard).
+- The [Kubernetes dashboard](https://kubernetes.prd.artsy.systems/) reflects the configuration, scale, or events
+  relating to specific system deployments.
+- See also other metrics and dashboards (AWS, MongoDB, Elasticsearch/kibana, Sendgrid...).
+
+If a mitigation (such as rolling back a release) requires some follow-up, make sure to note it on the incident and
+surface to the appropriate team. If a mitigation isn't immediately apparent, **escalate to an appropriate team or
+engineer**.
+
+#### Escalate
+
+When investigating an incident, you should involve the relevent team and stakeholders involved as soon as possible.
+**You are not expected to know the ins and outs of every system, so don't hesitate to involve the wider team.**
+
+<details>
+  <summary>To escalate an issue:</summary>
+
+- Identify the engineer responsible for the associated pull request or release. If not obvious, the
+  [affected system's local point person](https://www.notion.so/artsy/17c4b550458a4cb8bcbf1b68060d63e6?v=3604e2682d024b64bde705abb2facebd)
+  may be a good choice and can help investigate further.
+- [Add the responder](https://support.atlassian.com/opsgenie/docs/add-or-remove-a-responder/) to the incident as a
+  [Subject Matter Expert](https://support.atlassian.com/opsgenie/docs/define-incident-response-roles/). (This can
+  also be done directly from the OpsGenie-created Slack channel.) Adding the responder will notify them and keep
+  them alerted to ongoing developments.
+
+- Add the PM as a stakeholder to the issue. This will notify the stakeholder via email and allow them to receive
+  updates on the ongoing progress of the incident.
+
+  <img width=800 src="images/opsgenie-responding-0.png" />
+
+- Alert the relevent team via slack in the #inc-\* slack channel you created for the incident and share any
+  applicable work-arounds or talking points in the thread to unblock teammates or partners.
+  </details>
+
+#### Communicate
```

Comment:
It may be clearer if this section is organized around the three places the comms officer needs to be aware of/updating vs. just listing them in separate paragraphs:
1. The OpsGenie timeline
2. Slack: The original #incidents thread _and_ if discussion & resolution is happening in zoom, consider updating the #inc- channel as well with initial findings/explorations so people can follow async.
3. StatusPage